### PR TITLE
Include deploy/deployment-manager/ in shell scripts linting

### PR DESCRIPTION
**Pull request by romulets**
### Summary of your changes
Include deploy/deployment-manager/ scripts to shell linting

### Related Issues
- Fixes https://github.com/elastic/cloudbeat/issues/1364

### CODE REVIEW

File: deploy/deployment-manager/deploy.sh [hunk 1]
```diff
@@ -40,8 +40,11 @@ ELASTIC_ARTIFACT_SERVER=${ELASTIC_ARTIFACT_SERVER:-https://artifacts.elastic.co/
 DEPLOYMENT_LABELS=${DEPLOYMENT_LABELS:-type=cspm-gcp}
 
 # Set environment variables with the name and number of your project.
-export PROJECT_NAME=$(gcloud config get-value core/project)
-export PROJECT_NUMBER=$(gcloud projects list --filter=${PROJECT_NAME} --format="value(PROJECT_NUMBER)")
+PROJECT_NAME=$(gcloud config get-value core/project)
+export PROJECT_NAME
```
Comment (orestisfl) [inline deploy/deployment-manager/deploy.sh:44]: I wonder if these need to be exported at all?

File: deploy/deployment-manager/deploy.sh [hunk 2]
```diff
@@ -92,8 +95,10 @@ is_role_not_assigned() {
 }
 
 # Set environment variables with the name and number of your project.
-export PROJECT_NAME=$(gcloud config get-value core/project)
-export PROJECT_NUMBER=$(gcloud projects list --filter="${PROJECT_NAME}" --format="value(PROJECT_NUMBER)")
+PROJECT_NAME=$(gcloud config get-value core/project)
```
Comment (orestisfl) [inline deploy/deployment-manager/deploy.sh:98]: I wonder if these need to be exported at all?

**Comment by oren-zohar** [comment]
Should be covered by https://github.com/elastic/cloudbeat/pull/1568 but good initiative

File: deploy/deployment-manager/deploy.sh (hunk 1 above)
Comment (romulets) [reply deploy/deployment-manager/deploy.sh:44]: Maybe not. I don't know if this is being picked up somewhere else. I find a way to test this out

**Comment by romulets** [comment]
> Should be covered by #1568 but good initiative

@oren-zohar I see, I'll add a comment there to also add `deploy/deployment-manager` to the linter, but indeed the format does cover.

I'll close this one

**Pull request closed without merging**